src/__tests__/api.test.ts

```diff
@@ -157,4 +161,62 @@ describe("flashcard", () => {
       })
     })
   })
+  describe("deleting card", () => {
+    describe("delete card route", () => {
+      it("function deletes the requested flashcard if it was created less than 5 minutes ago and returns a status code of 204", async () => {
+        const idCardToDelete = new ObjectId(Date.now() - 5000).toString()
+        flashcardModel.create({
+          ...newCardMock,
+          createdAt: Date.now() - 5000,
+          _id: idCardToDelete,
+        })
+        let response
+
+        response = await request(app)
+          .delete(DEFAULT_ROUTE + idCardToDelete)
+          .set("Authorization", AUTHORIZATION_KEY)
+        expect(response.status).toBe(204)
+
+        response = await request(app)
+          .get(DEFAULT_ROUTE)
+          .set("Authorization", AUTHORIZATION_KEY)
+        const data: IFlashcard[] = response.body
+        expect(data.find((flashcard) => flashcard._id === idCardToDelete)).toBe(
+          undefined
+        )
+      })
+      it("function returns a status code of 403 if the flashcard was created more than 5 minutes ago", async () => {
+        const idCardToDelete = new ObjectId(
+          Date.now() - (1000 * 60 * 5 + 1)
+        ).toString()
+        flashcardModel.create({
+          ...newCardMock,
+          createdAt: Date.now() - (1000 * 60 * 5 + 1),
+          _id: idCardToDelete,
+        })
+        let response
+
+        response = await request(app)
+          .delete(DEFAULT_ROUTE + idCardToDelete)
+          .set("Authorization", AUTHORIZATION_KEY)
+        expect(response.status).toBe(403)
+
+        response = await request(app)
+          .get(DEFAULT_ROUTE)
+          .set("Authorization", AUTHORIZATION_KEY)
+        const data: IFlashcard[] = response.body
+        expect(
+          data.find((flashcard) => flashcard._id === idCardToDelete)
+        ).not.toBe(undefined)
```

Comment:
**nitpick:** This check is too weak - let's expect found card to have specific value, eg:

```ts
 expect(cardToDelete._id).toBe(idCardToDelete)
```